packaging/python/test_wheels.sh

```diff
@@ -116,13 +117,20 @@ test_wheel () {
     rm -rf tmp_mod x86_64
 }
 
-# setup python virtual environment
-venv_name="nrn_test_venv_${python_ver}"
-$python_exe -m venv $venv_name
-. $venv_name/bin/activate
+# creat python virtual environment and use `python` as binary name
+# because it will be correct one from venv.
+if [[ "$use_venv" != "false"* ]]; then
```

Comment:
to test the truthness of a variable I recently started using `[[ "$var" > 0 ]]` which evaluates to true in case var is not empty and not 0

Reply:
yeah, come across this `[[` recently. But for our case, if its empty, it won't match with `false`. So this would be ok no?